web/basics/using_bind_to_create_a_single_template_instance/index.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:polymer/polymer.dart';
```

Comment:
This demo can be deleted, in the new polymer auto binding templates should only be used outside of polymer elements https://www.polymer-project.org/1.0/docs/devguide/templates.html#dom-bind